testsuite/scripts/src/test/java/org/wildfly/scripts/test/StandaloneScriptTestCase.java

```diff
@@ -80,6 +87,15 @@ public void testBatchScriptJavaOptsEscaped() throws Exception {
         ServerConfigurator.removeJavaOpts(ServerHelper.JBOSS_HOME, STANDALONE_BASE_NAME, escapedVariable);
     }
 
+    @Test
+    public void testPowershellSupportedOnWindows() throws Exception {
+        // WFCORE-5917
+        // On Windows, command-line arguments of the process can be obtained only via PowerShell.
+        // For Windows, where PowerShell is not available, testScript WFCORE-5917 related assertions are skipped.
+        // The purpose of this test is to determine whether WFCORE-5917 has been verified.
+        Assume.assumeTrue(TestSuiteEnvironment.isWindows() && Shell.POWERSHELL.isSupported());
+    }
```

Comment:
Well, it will be more:
```
if (TestSuiteEnvironment.isWindows()) {
Assume.assumeTrue(Shell.POWERSHELL.isSupported())
}
```

So the test will report ignored only when the testsuite is running in Windows and PowerShell is not enabled. If the testsuite is running in Linux, we don't need to report WFCORE-5917 verification was ignored.